runtime/docker/volume.go

```diff
@@ -54,7 +55,18 @@ func (c *client) InspectVolume(ctx context.Context, b *pipeline.Build) ([]byte,
 		return nil, err
 	}
 
-	return []byte(v.Name + "\n"), nil
+	// convert volume type Volume to bytes with pretty print
+	//
+	// https://godoc.org/github.com/docker/docker/api/types#Volume
+	volume, err := json.MarshalIndent(v, "", " ")
+	if err != nil {
+		return nil, err
+	}
+
+	// add new line to end of bytes
+	volume = append(volume, "\n"...)
+
+	return volume, nil
```

Comment:
I think we could condense these lines into a single line 👍

Thoughts on that?

```suggestion
	// add new line to end of bytes
	return append(volume, "\n"...), nil
```